drivers/COMPONENT_WIFI_ESP32.lib

```diff
@@ -0,0 +1 @@
+https://github.com/ARMmbed/esp32-driver/#d6bd83f8710b4c475b3cce101d8b2c77e17feaf6
```

Comment:
Could you use here the COMPONENT format as has been done for the existing drivers?

https://github.com/ARMmbed/mbed-os-example-pelion/tree/master/drivers

Reply:
I am not clear about your comment, but the GR-LYCHEE has on-board ESP32 Wi-Fi module and it uses existing esp32-driver.

Reply:
Ah, sorry, I misread, you already did it as expected.